packages/website/src/components/contentful-page/helpers/page.jsx

```diff
@@ -0,0 +1,42 @@
+import { setupClient } from '../../../shared/contentful'
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer'
+
+export const getPageFromId = async (entryId) => {
+  const client = setupClient()
```

Comment:
We seem to set up a new client for every interaction with contentful - would it be better to have one for the whole app passed down through props instead?